aict_tools/scripts/train_dxdy_regressor.py

```diff
@@ -0,0 +1,165 @@
+import pandas as pd
+import click
+from sklearn import model_selection
+from sklearn import metrics
+from tqdm import tqdm
+import numpy as np
+
+from fact.io import write_data
+from ..io import save_model, read_telescope_data
+from ..preprocessing import (
+    convert_to_float32, calc_true_disp, convert_units, horizontal_to_camera,
+)
+from ..feature_generation import feature_generation
+from ..configuration import AICTConfig
+from ..logging import setup_logging
+
+
+@click.command()
+@click.argument('configuration_path', type=click.Path(exists=True, dir_okay=False))
+@click.argument('signal_path', type=click.Path(exists=True, dir_okay=False))
+@click.argument('predictions_path', type=click.Path(exists=False, dir_okay=False))
+@click.argument('dxdy_model_path', type=click.Path(exists=False, dir_okay=False))
+@click.option('-k', '--key', help='HDF5 key for h5py hdf5', default='events')
+@click.option('-v', '--verbose', help='Verbose log output', is_flag=True)
+def main(configuration_path, signal_path, predictions_path, dxdy_model_path, key, verbose):
+    '''
+    Train one learner to be able to reconstruct the source position.
+    One regressor for multiple outputs (dx,dy).
+
+    Both pmml and pickle format are supported for the output.
+
+    CONFIGURATION_PATH: Path to the config yaml file
+
+    SIGNAL_PATH: Path to the signal data
+
+    PREDICTIONS_PATH : path to the file where the mc predictions are stored.
+
+    DXDY_MODEL_PATH: Path to save the dxdy model to.
+
+        Allowed extensions are .pkl and .pmml.
+        If extension is .pmml, then both pmml and pkl file will be saved
```

Comment:
Actually we also support storing `.onnx` now, but I guess this is copy paste so wrong in all the help strings now?